routers/articles.js

```diff
@@ -162,12 +164,86 @@ router.get("/:slug", async (req, res) => {
     .populate("tagList")
     .exec()
 
-  const processedArticle = {
-    ...article.toObject(),
-    tagList: article.tagList.map((tag) => tag.name).sort(),
-    favorited: article.favoritedBy.includes(req.user?.userId),
+  let processedArticle = {}
+
+  if (article) {
+    processedArticle = {
+      ...article.toObject(),
+      tagList: article.tagList.map((tag) => tag.name).sort(),
+      favorited: article.favoritedBy.includes(req.user?.userId),
+    }
   }
   res.json({ article: processedArticle })
 })
 
+// Get comments
+router.get("/:slug/comments", async (req, res) => {
+  const { slug } = req.params
+  const comments = await Comment.find({ slug })
+
+  if (!comments) {
+    return res.status(404).json({ success: false, msg: "No comments found." })
+  }
+
+  res.json({ comments })
```

Comment:
Change `_id` to `id` before sending response?

Reply:
Fixed